README.md

```diff
@@ -2,6 +2,17 @@
 
 ## Usage
 
+Ensure docker is running or if you are using Boot2Docker, run:
+
+```console
+boot2docker up
+boot2docker shellinit
```

Comment:
My understanding about `shellinit` was that it outputs clear messaging about environment variables you should set in your shell profile, running what you suggest:

```
eval $(boot2docker shellinit)
```

Would make *this shell* work, but the user may not notice that they still need another step to make new shells work by default.